conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -677,7 +678,10 @@ def create_rebuild_graph(
         requirements = attrs.get("requirements", {})
         host = requirements.get("host", set())
         build = requirements.get("build", set())
-        bh = host or build
+        if include_build:
+            bh = host | build
```

Comment:
I think `build_host_requirements` or similar is a more descriptive variable name than `bh`.